packages/eslint-config/test/import/__snapshots__/import-unhappy.test.js.snap

```diff
@@ -0,0 +1,13 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`import / unhappy path unhappy 1`] = `
+"[31merror[39m: [1mDependency cycle detected[22m [2m(import/no-cycle)[22m at [32mpackages\\\\eslint-config\\\\test\\\\import\\\\__fixtures__\\\\import-unhappy.fixture.js:1:1[39m:
```

Comment:
<img width="1085" alt="image" src="https://user-images.githubusercontent.com/12021443/196972635-c17fbf39-1c6b-4c06-833e-4506edbfc1d9.png">

for example `/` instead of `\\\\`

Reply:
ah I'm blind, thanks! windows stuff....